core/integration/features/settlement/mark_to_market_interval_positions.feature

```diff
@@ -310,3 +310,136 @@ Feature: Check position tracking matches expected behaviour with MTM intervals.
     Then the parties should have the following account balances:
       | party           | asset | market id | margin | general |
       | designatedLoser | USD   | ETH/DEC20 | 0      | 0       |
+
+  Scenario: 003, settlement works correctly when party enters and leaves within one MTM window
+    Given the markets are updated:
+      | id        | linear slippage factor | quadratic slippage factor | lp price range |
+      | ETH/DEC19 | 1e0                    | 0                         | 2              |
+    And the parties deposit on asset's general account the following amount:
+      | party            | asset | amount        |
+      | aux              | USD   | 1000000000000 |
+      | aux2             | USD   | 1000000000000 |
+      | lp               | USD   | 1000000000000 |
+      | buyer            | USD   |       1000000 |
+      | seller           | USD   |       1000000 |
+      | party1           | USD   |       1000000 |
+      | party2           | USD   |       1000000 |
+    And the following network parameters are set:
+      | name                                    | value |
+      | network.markPriceUpdateMaximumFrequency | 10s   |
+
```

Comment:
Do you not need to set block time in second? markPriceUpdateMaximumFrequency is set in second

Reply:
What do you mean? The genesis time? It is whatever it is, the test only advances blocks (doesn't advance to any particular time) so it's fine.